src/Phinx/Console/Command/Create.php

```diff
@@ -131,7 +183,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
+        // Do we need to do the post creation call to the creation class?
+        if ($creationClassName) {
+            $creationClass->postMigrationCreation($filePath, $className, $this->getConfig()->getMigrationBaseClassName());
+        }
+
         $output->writeln('<info>using migration base class</info> ' . $classes['$useClassName']);
+
+        if (!empty($altTemplate)) {
+            $output->writeln('<info>using alternative template</info> ' . realpath($altTemplate));
```

Comment:
I wouldn't use `realpath` here, as it is not really necessary to show the full path. Reflecting the option the user entered should be enough.